README.md

```diff
@@ -11,18 +11,64 @@ terraform {
   required_providers {
     awsapigateway = {
       source  = "Traceableai/awsapigateway"
-      version = "~> 0.1.0"
+      version = "0.3.0"
     }
   }
 }
 
-provider "awsapigateway" {
-  region = "us-east-1"
+resource "awsapigateway_resouce" "traceable-example-2" {
+  identifier                 = uuid()
+  ignore_access_log_settings = false
+  account {
+    region                 = "us-east-1"
+    api_list               = ["api1", "api2"]
+    cross_account_role_arn = ""
+    exclude                = false
+  }
 }
```

Comment:
can add another one for include

Reply:
sure, sorry, just merged the pr :|

Reply:
@puneet-traceable https://github.com/Traceableai/terraform-provider-awsapigateway/pull/4